Source/WebKitLegacy/mac/WebView/WebPreferences.mm

```diff
@@ -3066,16 +3066,6 @@ - (void)setMediaRecorderEnabled:(BOOL)flag
     [self _setBoolValue:flag forKey:WebKitMediaRecorderEnabledPreferenceKey];
 }
 
-- (BOOL)CSSIndividualTransformPropertiesEnabled
-{
-    return [self _boolValueForKey:WebKitCSSIndividualTransformPropertiesEnabledPreferenceKey];
-}
-
-- (void)setCSSIndividualTransformPropertiesEnabled:(BOOL)flag
-{
-    [self _setBoolValue:flag forKey:WebKitCSSIndividualTransformPropertiesEnabledPreferenceKey];
-}
-
```

Comment:
You have to noop these and move them to the end of the file. Unless you want to deal with binary compatibility fallout as I understand it.

Reply:
See https://github.com/WebKit/WebKit/pull/20896 as an example of how bincompat can be preserved.

Reply:
I guess you could do a codesearch to see if anyone used them. I doubt it in this case, but you never know.